torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1841,7 +1857,7 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
            aten_name='linalg_multi_dot',
            skips=(
                # These call into check_alias_annotation which does not support TensorList inputs
-               SkipInfo('TestCommon', 'test_variant_consistency_jit', 
+               SkipInfo('TestCommon', 'test_variant_consistency_jit',
                         dtypes=[torch.float32, torch.int32]),
```

Comment:
Trimming the extra whitespace after the `SkipInfo` here causes the merge commit e1698c5 to contain the code change from `multi_dot`.